build.gradle

```diff
@@ -24,12 +24,12 @@ buildscript {
         maven { url "https://plugins.gradle.org/m2/" }
     }
     dependencies {
-        classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:3.0.3'
+        classpath 'com.netflix.nebula:gradle-extra-configurations-plugin:7.0.0'
```

Comment:
Update plugins to work with Java 17 and Gradle 7.3

Reply:
Wow, we were very far behind.  Do you recall what this plugin is used for?

Reply:
optional dependencies